README.md

```diff
@@ -176,8 +177,18 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 ### Automation
 
-- [Google Cloud Build](https://cloud.google.com/cloud-build/)
+- [GitHub Actions](https://github.com/features/actions/)
 
 ## Versioning
 
 This project adheres to [Semantic Versioning](http://semver.org) v2.
+
+**IMPORTANT** Changes that would require updates be made to the setup of
+projects that are using this package should me marked as `BREAKING CHANGE`.
+Doing so will configure the continuous delivery pipeline to release a new major
+version of the `@ridedott/eslint-config` package.
+
+Opposite to this, changes that require no updates be made to the setup of
+projects that are using this package will trigger the continuous delivery
+pipeline to release either a `minor` or `patch` version of
+`@ridedott/eslint-config`.
```

Comment:
This should be moved to Contributing. Here, towards consumers, we can phrase it a little different: `Changes and additions of linting rules are not considered breaking changes.`.